File: CS/dxSample/Form1.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace dxSample {
    public partial class Form1 : DevExpress.XtraEditors.XtraForm {
        public Form1() {
            InitializeComponent();
        }
        private void ShowPreview() {
            XtraReport1 report = new XtraReport1();
            report.CreateDocument();
            documentViewer1.DocumentSource = report;
        }

        private void Form1_Load(object sender, EventArgs e) {
            ShowPreview();
        }
    }
}

File: VB/dxSample/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq

Namespace dxSample
    Partial Public Class Form1
        Inherits DevExpress.XtraEditors.XtraForm

        Public Sub New()
            InitializeComponent()
        End Sub
        Private Sub ShowPreview()
            Dim report As New XtraReport1()
            report.CreateDocument()
            documentViewer1.DocumentSource = report
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
            ShowPreview()
        End Sub
    End Class
End Namespace
